# publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_base_01
  pid: {p: 70, i: 5, d: 10}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_01_02
  pid: {p: 70, i: 5, d: 10}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_02_03
  pid: {p: 70, i: 5, d: 10}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_03_04
  pid: {p: 70, i: 5, d: 10}
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_04_05
  pid: {p: 70, i: 5, d: 10}
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint_05_06
  pid: {p: 70, i: 5, d: 10}

 


